src/notyf/shared/notyf.service.ts

```diff
@@ -1,35 +1,36 @@
-import {
-  Injectable,
-  ApplicationRef,
-  Injector,
-  ComponentFactoryResolver,
-  EmbeddedViewRef,
-  ComponentFactory
-} from '@angular/core';
-import { ToastComponent, ToastType } from './../toast/toast.component';
+import { Injectable, Injector, ComponentFactoryResolver, EmbeddedViewRef } from '@angular/core';
+import { ToastComponent } from './../toast/toast.component';
```

Comment:
I guess you don't need the extra `./`. `../toast/toast.component` would be enough and clean.